assets/js/laws-regs-parser.js

```diff
@@ -56,6 +56,20 @@ export default function parseLawsAndRegs(mainContent) {
 
   parent.insertBefore(newMainContent, mainContent);
   mainContent.remove();
+  const toolBtn = document.querySelector('#crt-page--toolbutton');
+  const searchInput = document.querySelector('.searchbox');
+  if (toolBtn) {
+    const toolWrapper = document.querySelector('.tool-buttons');
+    toolBtn.addEventListener('click', () => {
+      toolWrapper.classList.add('active');
+      searchInput.focus();
+      toolBtn.classList.add('disabled');
+    });
+    const closeSearchBtn = toolWrapper.querySelector('#closeSearch');
+    closeSearchBtn.addEventListener('click', () => {
+      closeSearch(toolWrapper, searchInput);
+    });
```

Comment:
similar to setUpMobileSearch, `setUpSearchBox` would be a good helper function here to reduce the size of this setup code